lib/github/github/reviews.ex

```diff
@@ -8,16 +8,30 @@ defmodule BorsNG.GitHub.Reviews do
 
   @spec from_json!([map]) :: t
   def from_json!(json) do
-    json
+    reviews = json
     # Count only the latest review from a user,
     # by deduplicating using the user id
     |> Enum.reduce(%{}, fn
       %{"state" => "COMMENTED"}, acc ->
         acc
-      %{"user" => %{"id" => uid}, "state" => state}, acc ->
+      %{"user" => %{"login" => uid}, "state" => state}, acc ->
         Map.update(acc, uid, state, fn _ -> state end)
       end)
+
+    # Get the list of users who have approved this PR
+    approvers = reviews
+                |> Enum.flat_map(fn {_uid, state} ->
```

Comment:
If you are just getting one value per entry, then you don't need to flat-map, a filter -> map would be better:

```elixir
approved_by =
  reviews
  |> Enum.reject(fn {_, state} -> state == "DISMISSED" end)
  |> Enum.map(fn {username, _} -> username end)
  |> Enum.uniq()
```

Reply:
Also, considering that `reviews` is a map whose key is the uid, there's no way that an username/uid is repeated because a map imply in keys being unique, so the last operation is not necessary

Reply:
:zap: